libs/sdk-backend-base/src/dummyBackend/index.ts

```diff
@@ -457,6 +459,35 @@ class DummyOrganization implements IOrganization {
             getActiveTheme: () => Promise.resolve(undefined),
             setActiveTheme: () => Promise.resolve(),
             clearActiveTheme: () => Promise.resolve(),
+            createTheme: (theme: IThemeDefinition): Promise<IThemeMetadataObject> => {
+                return Promise.resolve({
+                    type: "theme",
+                    id: "theme_id",
+                    title: "Theme 1",
+                    description: "",
+                    production: true,
+                    deprecated: false,
+                    unlisted: false,
+                    ref: idRef("theme_id"),
+                    uri: "",
+                    theme: theme.theme,
+                });
+            },
+            updateTheme: (theme: IThemeDefinition): Promise<IThemeMetadataObject> => {
+                return Promise.resolve({
+                    type: "theme",
+                    id: "theme_id",
+                    title: "Theme 1",
+                    description: "",
+                    production: true,
+                    deprecated: false,
+                    unlisted: false,
+                    ref: idRef("theme_id"),
+                    uri: "theme_uri",
+                    theme: theme.theme,
+                });
+            },
+            deleteTheme: () => Promise.resolve(undefined),
```

Comment:
`undefined` here is unnecessary, right? Also in the recorded backend definition.

Reply:
fixed